portal-web/test/functional/com/liferay/portalweb/tests/enduseree/formsworkflow/metrics/WorkflowMetrics.testcase

```diff
@@ -98,6 +98,104 @@ definition {
 			value1 = "${key_testSLADefinitionDays}");
 	}
 
+	@description = "Apply filter for workload distribution for all assignees(LRQA-53632)"
+	@priority = "5"
+	test ApplyFilterForWorkloadDistribution {
+
+		ProductMenu.gotoPortlet(
+			category = "Configuration",
+			panel = "Site Administration",
+			portlet = "Workflow");
+
+		Workflow.configureWorkflow(
+			workflowDefinition = "Single Approver",
+			workflowResourceValue = "Blogs Entry");
+
+		ProductMenu.gotoPortlet(
+			category = "Workflow",
+			panel = "Control Panel",
+			portlet = "Metrics");
+
+		WorkflowMetrics.create6UsersAndAssign();
+
+		WorkflowMetrics.goToWorkflowProcessMetrics(workflowProcessName = "Single Approver");
+
+		Click(
+			key_assigneesNumber = "6",
+			locator1 = "WorkflowMetrics#METRICS_PERFORMANCE_BY_ASSIGNEE_CARD_VIEW_ALL_ASSIGNEES_LINK");
+
+		AssertElementPresent(locator1 = "WorkflowMetrics#METRICS_WORKLOAD_BY_ASSIGNEE_CARD_FILTER");
```

Comment:
Hi @calvinctvo ,
There is no need to assert this element because we are going to manipulate the filter later on the test. So, if the element is not present there, we will receive the same result.